yammbs/_minimize.py

```diff
@@ -181,6 +181,9 @@ def _run_openmm(
         except UnassignedValenceError:
             logger.warning(f"Skipping record {qcarchive_id} with unassigned valence terms")
             return None
+        except ValueError as e:  # charging error
+            logger.warning(f"Skipping record {qcarchive_id} with a value error: {e}")
```

Comment:
This is somewhat redundant here but could be useful to others in the future
```suggestion
            logger.warning(f"Skipping record {qcarchive_id} with a value error (probably a charge failure): {e}")
```